packages/react/src/primitives/Button/Button.tsx

```diff
@@ -24,14 +24,34 @@ const ButtonPrimitive: Primitive<ButtonProps, 'button'> = (
     size,
     type = 'button',
     variation,
+    colorTheme,
     ...rest
   },
   ref
 ) => {
+  // These variations support colorThemes. 'undefined' accounts for our
+  // 'default' variation which is not named. This is done so we do not need to
+  // include a breaking change of defining a default value for variation.
+  const supportsColorThemes = ['link', 'primary', undefined].includes(
+    variation
+  );
```

Comment:
Nit: not a fan of "magic values", would prefer having a const defined outside the component, i.e.

```
const supportedVariations = ['link', 'primary', undefined];
```

Reply:
Thanks, updated in latest!